### PR TITLE
Fixed Saxml deployment documentation after ConfigMap change. 

### PR DESCRIPTION
**Pull request by Edwinhr716**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes

Fixes #

#### Special notes for your reviewer
If model attribute isn't set, model will default to the last element of the modelpath all in lowercase. So in the example, the correct model to pass for the API call is lmcloudspmd175b32test.

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by liurupeng** [comment]
could you rebase to include the test fix? thanks! @Edwinhr716

**Comment by Edwinhr716** [comment]
rebased

**Comment by liurupeng** [comment]
thanks! @Edwinhr716

File: docs/example/saxml/README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ curl --request POST \
  --header "Content-type: application/json" \
 -s ${LB_IP}:${PORT}/generate --data \
 '{
-  "model": "/sax/test/spmd",
+  "model": "/sax/test/lmcloudspmd175b32test",
```
[the remark this answers is not in the document]
Comment (ahg-g) [reply docs/example/saxml/README.md:97]: do we need to update this now that it is /sax/cell?

**Comment by Edwinhr716** [comment]
Changed

**Pull request merged**